src/frontend/apps/desk/src/features/teams/components/ModalUpdateTeam.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Button,
+  Input,
+  Loader,
+  Modal,
+  ModalSize,
+  VariantType,
+  useToastProvider,
+} from '@openfun/cunningham-react';
+import { t } from 'i18next';
+import { useState } from 'react';
+
+import { Box, Text, TextErrors } from '@/components';
+import useCunninghamTheme from '@/cunningham/useCunninghamTheme';
+
+import { Team, useUpdateTeam } from '../api';
+import IconEdit from '../assets/icon-edit.svg';
+
+interface ModalUpdateTeamProps {
+  onClose: () => void;
+  team: Team;
+}
+
+export const ModalUpdateTeam = ({ onClose, team }: ModalUpdateTeamProps) => {
+  const { colorsTokens } = useCunninghamTheme();
+  const [newTeamName, setNewTeamName] = useState('');
+  const { toast } = useToastProvider();
+  const {
+    mutate: updateTeam,
+    isError,
+    isPending,
+    error,
+  } = useUpdateTeam({
+    onSuccess: () => {
+      toast(t('The team has been updated.'), VariantType.SUCCESS, {
+        duration: 4000,
+      });
+      onClose();
+    },
+  });
+
+  return (
+    <Modal
```

Comment:
Modal's background color should be white

Reply:
[💄(app-desk) update ui](https://github.com/numerique-gouv/people/pull/136/commits/b93850c01e120f24049021024b545d38e139e820)